server.js

```diff
@@ -928,6 +928,16 @@ module.exports.initExpress = function () {
       require('./pages/generatedFilesQuestion/generatedFilesQuestion'),
     ]
   );
+
+  // Submission files
+  app.use(
+    '/pl/course_instance/:course_instance_id/instructor/instance_question/:instance_question_id/submission/:submission_id/files',
```

Comment:
I think `.../file` might be more idiomatic than `/files`? We tend to use singular nouns in most of our routes.

Reply:
Changed to `/file` in ac775c4.